src/main/java/dev/jbang/cli/Run.java

```diff
@@ -33,6 +30,10 @@
 			"--main" }, description = "Main class to use when running. Used primarily for running jar's.")
 	String main;
 
+	@CommandLine.Option(names = { "-v",
```

Comment:
I kinda always felt the version used should be coming from the script, i.e. `//JAVA_VERSION 11+` - and it should be able to say 11+ meaning if you have 14 that is fine. If you say 11 it must be 11. that kinda thing.

This flag would be an override of that setting - and as such I would say it should be named something like `-j/--java 11`

Reply:
Yes, I agree, the setting was just the quickest to add, I needed something to test with :-)

And being able to say "X+" had occurred to me too, but I went for the easier implementation first. Now that I know you want it that way I'll change it.

`-j/--java` sure.